jenkins-pipelines/linux_desktop_build.groovy

```diff
@@ -37,7 +37,7 @@ pipeline {
     }
     stage("Integration Tests") {
       environment {
-        FC_DOCKER_IMAGE = credentials("docker-latest-fc")
+        AWS_DOMAIN = credentials("docker-aws-domain-conservator")
       }
```

Comment:
FYI you can derive the domain from the credentials so it doesn't need to be a separate secret

    aws ecr get-login --no-include-email --region us-east-1 | sed "s~.*https://~~"
    blahblah.us-east-1.amazonaws.com

Reply:
I don't know how to export a variable from one stage to another, and domain is used in several stages

Reply:
Exporting between stages does look really clunky in jenkins